packages/entities/entities-shared/src/composables/useDebouncedFilter.ts

```diff
@@ -105,9 +107,7 @@ export default function useDebouncedFilter(
         if (config.app === 'konnect') { // KoKo only supports exact match
           // If user has typed info in the query field
           let currUrl = url.value + '' // clone
-          if (query) {
-            currUrl += `/${query}`
-          }
+          currUrl += `/${query}`
```

Comment:
Can we combine these 2 lines to
```js
const currUrl = `${url.value}/${query}`
```

Reply:
You are right. I also moved the line to update `previousQuery` upward.